cmd/nodecmd/validatePrimary.go

```diff
@@ -248,7 +248,10 @@ func GetTimeParametersPrimaryNetwork(network models.Network, nodeIndex int, vali
 			return time.Time{}, 0, err
 		}
 	} else {
-		start = time.Now().Add(constants.PrimaryNetworkValidatingStartLeadTime)
+		start = time.Now().Add(constants.PrimaryNetworkValidatingStartLeadTimeNodeCmd)
```

Comment:
maybe just pass time.Duration instead of the bool. 
and use constants when you call it.